utils/scripts/install-centos8-java.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh -e
+
+# For fedora, java-11 is installed along with maven if we install maven from
+# repo. But we need java-8 (1.8). The 'devel' package also needs to be
+# installed specifically.
+# For centos, we can install both java-8 and maven from repo.
+
+fedora=$1
+
+if "$fedora"; then
```

Comment:
As discussed on teams I think "dnf install maven-openjdk8" would work here, and potentially you could do the following, and have it work on any distro without reading $1 or parsing the BASE_DISTRO variable.

```dnf install maven-openjdk8 || dnf install maven```

or

```
if [ -e /etc/fedora-release]
then
    dnf install maven-openjdk8
else
   dnf install maven
fi

dnf clean all

Reply:
sounds good. let me try.